CONTRIBUTING.md

```diff
@@ -48,11 +48,13 @@ GitHub, clone, and develop on a branch. Steps:
    $ git push -u origin my-feature
    ```
 
+  *Note*: if your PR includes changes to files under the ``doc/`` or ``examples/`` folders, include in your commit message ``[doc build]``. Otherwise, include in your commit message ``[doc skip]``. This will allow the CI to avoid unnecessarily building the docs.
```

Comment:
Everything else seems good. Just needs a few formatting corrections:

- I think the *Note* may not be needed
- *If your PR
- include ``[doc build]`` in your commit message
- Otherwise, include ``[doc skip]`` in your commit message